src/dynamic.ts

```diff
@@ -30,22 +32,27 @@ export class Dynamic extends Command {
 
     public async run(args: string[]): Promise<Command> {
             try {
-                args = args.slice(2);
-
-                while(args.length>0){
-                    let command = this.getCommand(args);
-                    let params = command.parseOptions(args);
-                    let paramCount = (Object.entries(params).length) + (Object.values(params).length);
-                    if(args.length%2==1){
-                        console.error(`\n${Color.FgRed}Error${Color.Reset}`, 'Options formatting');
-                        break;
+                let parsedArgs = args.slice(2);
+                let commandIndex = [];
+                let specialCmds = [];
+                for(let i=0;i<parsedArgs.length;i++){
+                    for(let cmd in this.commands){
+                        if(parsedArgs[i] === this.commands[cmd].name){
+                            commandIndex.push(i)
+                        }
+                    }
+                    if(commandIndex.length==0){
+                        specialCmds.push(parsedArgs[i]);
+                    }
+                }
+                for(let i=0;i<commandIndex.length;i++){
```

Comment:
You may not need the logic to avoid the overflow
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice